File: VB/WindowsApplication1/MyLayoutControl/MySkinTabHeaderViewInfo.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms
Imports DevExpress.XtraPrinting.Export
Imports DevExpress.XtraTab.Registrator
Imports DevExpress.XtraTab.ViewInfo

Namespace WindowsApplication1
	Public Class MySkinTabHeaderViewInfo
		Inherits SkinTabHeaderViewInfo
		Public Sub New(ByVal viewInfo As BaseTabControlViewInfo)
			MyBase.New(viewInfo)

		End Sub

	Public Overrides Function CalcMinPageSize(ByVal info As BaseTabPageViewInfo) As Size
		Return MyPageMinSizeHelper.PageMinimumSize
	End Function

	End Class
End Namespace

File: CS/Demo/WindowsApplication1/MyLayoutControl/MySkinTabHeaderViewInfo.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using DevExpress.XtraPrinting.Export;
using DevExpress.XtraTab.Registrator;
using DevExpress.XtraTab.ViewInfo;

namespace WindowsApplication1
{
    public class MySkinTabHeaderViewInfo : SkinTabHeaderViewInfo
    {
        public MySkinTabHeaderViewInfo(BaseTabControlViewInfo viewInfo)
            : base(viewInfo)
        {

        }
        public override Size CalcMinPageSize(BaseTabPageViewInfo info)
        {
            return MyPageMinSizeHelper.PageMinimumSize;
        }
    }
}
